### PR TITLE
prepare release for build/ => build-tools/ rename in main repo

### PR DESCRIPTION
**Pull request by mikedanese**
@ixdy @spxtr

### CODE REVIEW

**Comment by spxtr** [comment]
It looks fine but I might appreciate a comment saying why this is the way it is, and maybe a link back to the issue.

**Comment by mikedanese** [comment]
@spxtr done.

File: branchff [hunk 1]
```diff
@@ -140,7 +140,15 @@ logrun -s hack/update-all.sh
 
 # Redo the branching activities
 logecho -n "Check/rerun versionize-docs.sh again: "
-logrun -s build/versionize-docs.sh $RELEASE_BRANCH
+# TODO: remove once we don't support k8s versions with build/ anymore
```
Comment (david-mcmahon) [inline branchff]: Same change here as other comment.

File: push-build.sh [hunk 2]
```diff
@@ -179,7 +179,15 @@ if ((FLAGS_federation)); then
   ############################################################################
   logecho -n "Push federation images: "
   # FEDERATION_PUSH_REPO_BASE should be set by the calling job (yaml)
-  logrun -s ${KUBE_ROOT}/build/push-federation-images.sh
+  # TODO: remove once we don't support k8s versions with build/ anymore
```
Comment (david-mcmahon) [inline push-build.sh]: Same change here as other comment.

**Comment by mikedanese** [comment]
@david-mcmahon fixed.

**Comment by ixdy** [comment]
It's orthogonal to the bazel effort, but is there any reason we shouldn't just bring `versionize-docs.sh` and `push-federation-images.sh` over to this repo?

**Comment by david-mcmahon** [comment]
@ixdy Yes those could come over.  I had started working on pulling `push-federation-images.sh` over during the initial transition, but the tentacles were deep for that one and I put it off til 'later'.

**Pull request merged**